.github/workflows/stale.yml

```diff
@@ -0,0 +1,24 @@
+name: Mark stale issues and pull requests
+
+on:
+  schedule:
+  - cron: "30 1 * * *"
+
+jobs:
+  stale:
+
+    runs-on: ubuntu-latest
+
+    steps:
+    - uses: actions/stale@v3
+      with:
+        repo-token: ${{ secrets.GITHUB_TOKEN }}
+        stale-issue-message: 'This issue has been automatically marked as stale because it has not had recent activity. It will be closed if no further activity occurs.'
+        stale-pr-message: 'This pull request has been automatically marked as stale because it has not had recent activity. It will be closed if no further activity occurs.'
```

Comment:
Just to make sure I'm reading this correctly: the message says it will be closed if no action takes place, but it looks like this mechanism will not automatically close it. Is that right? I assume the -1 values for days before close mean that it will not be closed by this system. I suppose the intention is that things marked stale will be manually triaged and potentially closed if deemed appropriate.

We might want to re-word the issue message to express our intention. When I submit a PR, I generally have responsibility to shepherd my PR to completion. For instance, I currently have an old PR that's relatively low priority for the project and myself, but it will have marginal value and I intend to get it done eventually. But the fact that it is stale by these standards is my responsibility as the submitter of the PR. If I get this message on my PR, I will feel nudged to do something with it, and that seems good. I should take some action.

I wonder, on the other hand, how someone will feel if an issue they filed gets this message. To them it was probably an annoyance or something wrong enough such that they felt a responsibility to raise the issue to the community. They may be a user rather than a developer, and getting a message saying that their issue will be closed if there is no activity may make them feel a bit miffed: there's not much they can do to resolve the issue. Perhaps they can re-articulate things, but beyond that they might feel helpless. 

Perhaps something like:

'This issue has been automatically marked as stale because it has not had recent activity. Marking it stale to flag it for further consideration by the community."

Reply:
My thought is we would automatically close PRs and issues when there has been no activity.  I didn't want to do that yet and just want to label them Stale and make a comment for people to possibly updated the PR and Issue.

It is more about testing this out since we don't have any actions, yet, that automatically edit PRs and issues.

Reply:
Understood. You might want to consider my comment about the wording of the issue message though and what that may communicate to the submitter. Unless my understanding of the intention is wrong.